zmq-deferred/src/jbuild

```diff
@@ -0,0 +1,6 @@
+(jbuild_version 1)
+
+(library
+ ((name zmq_deferred)
+  (wrapped true)
+  (libraries (zmq base))))
```

Comment:
Can't we drop base here? If we're able to do that, we might just be able to include this module in `zmq` itself. Also taking the opportunity to make it build with `wrapped true` as well. In general, I don't see this library as useful on its own, so I'd like to look for ways to avoid having it visible to the user.

P.S. this will build locally, but it will actually install a broken META file (dune bug) because `zmq_deferred` also needs a `public_name`. I assume you just omitted that by accident?

Reply:
I've given zmq-deferred the publicname zmq.deferred, which makes it part of zmq